tensorboard/plugins/histogram/README.md

```diff
@@ -4,6 +4,8 @@ The TensorBoard Histogram Dashboard displays how the distribution of some
 `Tensor` in your TensorFlow graph has changed over time. It does this by showing
 many histograms visualizations of your tensor at different points in time.
 
+Note that due to frontend limitation and for visualization purpose, by default the histogram data would be rebinned into 30 bins using linear binning, where `y` counts from the original input bins are distributed among the new (increasing and non-overlapping) bins based on the amount of overlap (see existing [issue](https://github.com/tensorflow/tensorboard/issues/1803)).
```

Comment:
Hmm, I'd skip "frontend limitation" (since we _could_ change this code) and "by default" (since there's no way to disable it), and maybe add some explicit acknowledgement that this can lead to distortions or inaccuracy?  E.g. something like:

> Note that for visualization purposes, histogram data is rebinned into 30 linearly spaced bins, where `y` counts from the original input bins at each step are linearly distributed into the new bins, which are shared across all steps. For some distributions, this can result in significant distortions (see existing [issue]).

Reply:
Modified. Thanks!